### DISCUSSION TITLE
Issue with non-installed dependency (Firebase AdmiN SDK)

### DISCUSSION THREAD

**Discussion by YsarocK** [vercel/community#930] [Help]
Hello,

I built an app with Nuxt 3 (RC11) and one API route using the firebase-admin package.
The local build works perfectly but when deploying to Vercel, it show this error :

`{"url":"/api/event/theID","statusCode":500,"statusMessage":"Internal Server Error","message":"Cannot find module 'protobufjs/minimal'\nRequire stack:\n- /var/task/node_modules/google-gax/build/protos/iam_service.js\n- /var/task/node_modules/google-gax/build/src/index.js\n- /var/task/node_modules/@google-cloud/firestore/build/src/index.js\n- /var/task/node_modules/firebase-admin/lib/firestore/index.js","stack":""}`

**I have tried those things** : 
- `npm update`
-  Installed & reinstalled firebase-admin package
-  Forced `protobufjs` in package.json

Currently I'm blocked :/

> **Small update** :
> Tried to deploy the exact same app withe exact same config to Netlify, works like a charm.
> Interested to know what is the error caused by Vercel.
> 

Thanks a lot !

**Comment by amyegan** [comment]
Sorry you're running into this error @YsarocK. Are you able to share your repo or a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) to help me investigate?
